spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -27,6 +27,8 @@ def assert_ems_counts
     expect(ems.miq_templates.count).to eq(57)
     expect(ems.key_pairs.count).to eq(2)
     expect(ems.availability_zones.count).to eq(3)
+    expect(ems.resource_groups.count).to eq(3)
+    expect(ems.resource_groups.first.name).to eq('camc-test')
```

Comment:
This specific spec is just checking counts, we should add an `assert_specific_resource_group` test that checks the attributes are set properly.  You can check the `:name` here along with the `:type` to ensure that the core `model_class` is setting the expected subclass string.

Reply:
Also `.first` on an association that has >1 entry can cause sporadic test failures, should be doing `.find_by(:ems_ref => "something")`

Reply:
I am using `name`, because that is the pattern that was checked-in originally.